config/deploy/docker/assets/Dockerfile.open-path-warehouse.base

```diff
@@ -71,6 +71,13 @@ RUN chmod +x /usr/bin/entrypoint.sh \
   && rm .env \
   && mkdir tmp/pids
 
+RUN chown -R app:app tmp
+RUN touch /etc/timezone
+RUN ln -sf /app/etc-localtime /etc/localtime
```

Comment:
@yakloinsteak This part is a little sketch; the entrypoint now does `cp /usr/share/zoneinfo/$TIMEZONE /app/etc-localtime`. This allows us to set `/etc/localtime` as non-root (trying to fix permissions on that file to allow direct `cp` proved fruitless). Wanted to draw your attention here since I'm not 100% convinced this is kosher.

Reply:
Can you temporarily set USER back to root and then set it back after? As long as the app is running at non-root, it's fine I think.

Reply:
Well the permissions issue isn't this line in the Dockerfile, it's in the `cp` that happens in the entrypoint. And I think the `cp /usr/share/zoneinfo/$TIMEZONE /etc/localtime` HAS to happen in the entrypoint because we don't have the application environment with `$TIMEZONE` set when the image is built.

Reply:
Oh, sorry. I missed that. I'm with you now. So here's another silly idea. Can you install sudo and allow a command that does what you need? I don't love that idea either. 

Perhaps we can just use utc or eastern. The app sets the timezone as well, so I'm not sure what really depends on it being anything in particular.

Reply:
I'm pretty sure we _do_ need the timezone set at the "server" level.  I don't remember why, but it has been a real pain to get everything into the right timezones.